doc/api.yml

```diff
@@ -2007,6 +2007,16 @@ paths:
               - UNPAID
               - LATE
               - PAID
+        - name: start_month
+          in: query
+          schema:
+            type: string
+            format: date
+        - name: end_month
```

Comment:
month_to (same here)

Reply:
here we don't need to specify the month, we will just return all by default with pagination